ArjunaJTS/integration/src/main/java/com/arjuna/ats/jbossatx/BaseTransactionManagerDelegate.java

```diff
@@ -198,6 +205,12 @@ public Transaction suspend()
     // return all the event listeners associated with this thread
     private Collection<TransactionListener> getListeners(Transaction transaction, boolean create)
     {
+        if(!LISTENERS_ENABLED) {
+            if(create) {
+                throw new IllegalStateException("Transaction listeners are disabled and should not be used. If you need them they can be enabled via -D" + LISTENERS_ENABLED_PROP + "=true");
```

Comment:
@mmusgrov I assume the exception message should be internationalized. Will you create a PR for that or should do I?